types/office-js-preview/index.d.ts

```diff
@@ -437,7 +437,7 @@ declare namespace Office {
         */
         asyncContext: any;
         /**
-        * Gets an object that may provide additional information if an error occurred.
+        * Gets an object that may provide additional information if an {@link Office.Error | error} occurred.
```

Comment:
This is a larger issue, but we should change "may provide" to "provides". Not sure why we're trying to weasel our way out of helping developers 😉.

Reply:
Lol. I used "may provide" because at this point we don't give additional information for all errors.